src/update_metadata.rs

```diff
@@ -25,6 +25,32 @@ use crate::limiter::create_rate_limiter;
 use crate::parse::{convert_local_to_remote_data, parse_cli_creators, parse_keypair};
 use crate::{constants::*, parse::parse_solana_config};
 
+pub fn update_seller_fee_basis_points_one(
+    client: &RpcClient,
+    keypair: Option<String>,
+    mint_account: &str,
+    new_seller_fee_basis_points: &u16,
```

Comment:
Nit: this doesn't need to be a reference as Rust primitives are stack copied when needed.

Reply:
Sorry, I don't understand. Are you saying don't use & in function parameters?

Reply:
Yeah the `&` shouldn't be necessary, but it's not a big deal.